app/models/visit.js

```diff
@@ -36,77 +36,75 @@ function paymentStateAcceptance(object) {
 }
 
 export default AbstractModel.extend({
-  additionalDiagnoses: DS.attr(), // Yes, the plural of diagnosis is diagnoses!
-  charges: DS.hasMany('proc-charge', {
-    async: false
-  }),
+  // Attributes
+  additionalDiagnoses: DS.attr(),
   customForms: DS.attr('custom-forms'),
-  diagnoses: DS.hasMany('diagnosis'),
   dischargeInfo: DS.attr('string'),
   endDate: DS.attr('date'), // if visit type is outpatient, startDate and endDate are equal
   examiner: DS.attr('string'),
   hasAppointment: DS.attr('boolean', { defaultValue: false }),
   history: DS.attr('string'), // No longer used
   historySince: DS.attr('string'), // History of the Present Illness -- no longer used
-  imaging: DS.hasMany('imaging', { async: true }),
-  labs: DS.hasMany('lab', { async: true }),
   location: DS.attr('string'),
-  medication: DS.hasMany('medication', { async: true }),
-  // this field is being deprecated in favor of patient-note
-  notes: DS.attr('string'),
-  patientNotes: DS.hasMany('patient-note', { async: true }),
+  notes: DS.attr('string'), // this field is being deprecated in favor of patient-note
   outPatient: DS.attr('boolean'),
-  patient: DS.belongsTo('patient', {
-    async: false
-  }),
   paymentState: DS.attr('string', { defaultValue: PAYMENT_STATES.PENDING }),
   primaryDiagnosis: DS.attr('string'), // No longer used -- diagnoses are stored in diagnoses hasMany relationship
   primaryBillingDiagnosis: DS.attr('string'), // AKA final diagnosis
   primaryBillingDiagnosisId: DS.attr('string'),
-  procedures: DS.hasMany('procedure', { async: true }),
   reasonForVisit: DS.attr('string'),
   startDate: DS.attr('date'),
   status: DS.attr('string'),
   visitType: DS.attr(),
+
+  // Associations
+  charges: DS.hasMany('proc-charge', { async: false }),
+  diagnoses: DS.hasMany('diagnosis'),
+  imaging: DS.hasMany('imaging', { async: true }),
+  labs: DS.hasMany('lab', { async: true }),
+  medication: DS.hasMany('medication', { async: true }),
+  patient: DS.belongsTo('patient', { async: false }),
+  patientNotes: DS.hasMany('patient-note', { async: true }),
+  procedures: DS.hasMany('procedure', { async: true }),
   vitals: DS.hasMany('vital', { async: true }),
 
   diagnosisList: computed('diagnoses.[]', function() {
-    let diagnoses = this.get('diagnoses');
+    let diagnoses = get(this, 'diagnoses');
     let diagnosisList = diagnoses.map((diagnosis) => {
       return diagnosis.get('diagnosis');
     });
     return diagnosisList;
   }),
 
   hasAppointmentLabel: computed('hasAppointment', function() {
-    let hasAppointment = this.get('hasAppointment');
-    let i18n = this.get('i18n');
-    if (hasAppointment === true) {
+    let hasAppointment = get(this, 'hasAppointment');
+    let i18n = get(this, 'i18n');
+    if (hasAppointment) {
```

Comment:
This line should remain as it was before because 
```js
if (hasAppointment === true) {
```
Only returns true if and only if hasAppointment has a value of true
```js
if (hasAppointment) {
```
Returns true even if hasAppointment has a value other than true (eg hasAppointment = 'N' would return true)